components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -1,127 +0,0 @@
-import { getLocale } from '../api/localeAPI'
-import {
-  addSiteCosmeticFilter,
-  removeSiteFilter,
-  removeAllFilters
-} from '../api/cosmeticFilterAPI'
-import shieldsPanelActions from '../actions/shieldsPanelActions'
-
-export let rule = {
-  host: '',
-  selector: ''
-}
-
-// parent menu
-chrome.contextMenus.create({
-  title: 'Brave',
-  id: 'brave',
-  contexts: ['all']
-})
-// block ad child menu
-chrome.contextMenus.create({
-  title: getLocale('addBlockElement'),
-  id: 'addBlockElement',
-  parentId: 'brave',
-  contexts: ['all']
-})
-chrome.contextMenus.create({
-  title: getLocale('resetSiteFilterSettings'),
-  id: 'resetSiteFilterSettings',
-  parentId: 'brave',
-  contexts: ['all']
-})
-chrome.contextMenus.create({
-  title: getLocale('resetAllFilterSettings'),
-  id: 'resetAllFilterSettings',
-  parentId: 'brave',
-  contexts: ['all']
-})
-// context menu listener emit event -> query -> tabsCallback -> onSelectorReturned
-
-chrome.contextMenus.onClicked.addListener((info: chrome.contextMenus.OnClickData, tab: chrome.tabs.Tab) => {
-  onContextMenuClicked(info, tab)
-})
-
-// content script listener for right click DOM selection event
-chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
-  const action = typeof msg === 'string' ? msg : msg.type
-  switch (action) {
-    case 'contextMenuOpened': {
-      rule.host = msg.baseURI
-      break
-    }
-    case 'hiddenClassIdSelectors': {
-      const tab = sender.tab
-      if (tab === undefined) {
-        break
-      }
-      const tabId = tab.id
-      if (tabId === undefined) {
-        break
-      }
-      shieldsPanelActions.generateClassIdStylesheet(tabId, msg.classes, msg.ids)
-      break
-    }
-    case 'contentScriptsLoaded': {
-      const tab = sender.tab
-      if (tab === undefined) {
-        break
-      }
-      const tabId = tab.id
-      if (tabId === undefined) {
-        break
-      }
-      const url = tab.url
-      if (url === undefined) {
-        break
-      }
-      shieldsPanelActions.contentScriptsLoaded(tabId, url)
-    }
-  }
-})
```

Comment:
Oops, I never updated the comment. We'll still need the `'hiddenClassIdSelectors'` and `'contentScriptsLoaded'` cases. `'contextMenuOpened'` can go.

Everything else in this file can still be deleted safely.

Reply:
Added back! 😄 👍